python/procstar/agent/conn.py

```diff
@@ -10,6 +10,8 @@
 import logging
 import random
 import time
+import websockets.protocol
+from   websockets.asyncio.server import ServerConnection
```

Comment:
nit: I think it's slightly better to import in the following way, just in case the internals change at some point:

```suggestion
from   websockets import ServerConnection, State
```